common/djangoapps/third_party_auth/saml.py

```diff
@@ -95,6 +95,16 @@ def wrapped_method(*args, **kwargs):
 
         return auth_inst
 
+    def auth_complete(self, *args, **kwargs):
+        """
+        The user has been redirected back from the IdP and we should
+        now log them in, if everything checks out.
+        """
+        try:
+            return super(SAMLAuthBackend, self).auth_complete(*args, **kwargs)
+        except Exception as ex:
+            log.exception('auth_complete failed: An exception occured. Message "%s"', ex.message)
```

Comment:
We should be logging the request data here.

Reply:
I added the logs to request_data [HERE](https://github.com/edx/edx-platform/pull/15972/files#diff-a127b8f7fe6281f8baa46c11d1e02e84R69)